massa-models/src/amount.rs

```diff
@@ -106,6 +108,11 @@ impl Amount {
         Amount::from_decimal(res)
     }
 
+    /// Return the representation of the amount as a mantissa and a scale.
+    pub fn to_mantissa_scale(&self) -> (u64, u32) {
```

Comment:
@aoudiamoncef 
I also needed this function so i copied/pasted what you did in the issue.
I'm not sure about the file where you wanted to put the constants ?